phpBB/styles/prosilver/template/bbcode.html

```diff
@@ -70,7 +70,7 @@
 
 <!-- BEGIN size --><span style="font-size: {SIZE}%; line-height: 116%;">{TEXT}</span><!-- END size -->
 
-<!-- BEGIN img --><img src="{URL}" class="postimage" alt="{L_IMAGE}" /><!-- END img -->
+<!-- BEGIN img --><img class="postimage" src="{{ URL }}" data-src="{{ T_THEME_PATH }}/svg/three-dots.svg" alt="{{ lang('IMAGE') }}" ><!-- END img -->
```

Comment:
BBCode templates are really XSL. If you don't close void elements with `/>` they'll be loaded as HTML then converted to XSL. There's no support for Twig functions. `{{ URL }}` will probably not do anything useful. `{$T_THEME_PATH}` can be used if the parameter is set. https://github.com/phpbb/phpbb/blob/9df210b884785ec1e62449b0f8bd337915df2e03/phpBB/phpbb/textformatter/s9e/renderer.php#L176-L182